### PR TITLE
windows_winusb: Add support for reporting super speed plus devices

### PR DESCRIPTION
**Pull request by hjmallon**
Tested with USB-C 3.1 device on Windows 10 latest with Dell Laptop.

### CODE REVIEW

File: libusb/os/windows_winusb.c [hunk 1]
```diff
@@ -898,6 +900,7 @@ static int init_device(struct libusb_device *dev, struct libusb_device *parent_d
 		case 1: dev->speed = LIBUSB_SPEED_FULL; break;
 		case 2: dev->speed = LIBUSB_SPEED_HIGH; break;
 		case 3: dev->speed = LIBUSB_SPEED_SUPER; break;
+		case 4: dev->speed = LIBUSB_SPEED_SUPER_PLUS; break;
```
Comment (philemonf) [inline libusb/os/windows_winusb.c]: There are some checks in the code that checks for speed == LIBUSB_SPEED_SUPER. You might want to make sure that they also check for LIBUSB_SPEED_SUPER where this applies.

Comment (philemonf) [reply libusb/os/windows_winusb.c]: I know that libusb_get_max_iso_packet_size is doing it for instance. There might be other places.

Comment (philemonf) [reply libusb/os/windows_winusb.c]: PS: l am not a maintainer, so you might want to wait to get a feedback from them.

**Comment by hjelmn** [comment]
I have no way to check if this does what it says it does. If anyone else can verify that would be helpful.

File: libusb/os/windows_winusb.c (hunk 1 above)
Comment (hjmallon) [reply libusb/os/windows_winusb.c]: I have added that as a separate possible bug. It is not caused by this change. https://github.com/libusb/libusb/issues/553

**Comment by hjmallon** [comment]
@hjelmn is there anyone about here who can confirm with windows?

Maybe (sorry for tagging - I just pulled names from recent windows commits):
@gogoer , @dmitrykos , @nxpfrankli , @DjArt , @dickens ?

**Comment by nxpfrankli** [comment]
code itself look good. but I have not device to test it.

**Comment by dmitrykos** [comment]
@hjelmn, unfortunately do not have super-speed USB device to test.

File: libusb/os/windows_winusb.c (hunk 1 above)
Comment (philemonf) [reply libusb/os/windows_winusb.c]: It's probably just a matter of changing the == to >= if the spec for isochronous packet size has not changed in SuperSpeed Plus (I don't know if it has).

**Pull request closed without merging**